plugins/zenoh-plugin-storage-manager/src/replica/align_queryable.rs

```diff
@@ -212,46 +212,36 @@ impl AlignQueryable {
 
 // replying queries
 impl AlignQueryable {
-    async fn get_entry_with_ts(&self, timestamp: Timestamp) -> Option<Sample> {
+    async fn get_entry(&self, logentry: LogEntry) -> Option<Sample> {
```

Comment:
Passing `logentry` as a ref (`&LogEntry`) is sufficient and will avoid to clone it at line 163